Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM redwolfgang20/grails as builder
```

Comment:
This makes me nervous. It's entry in dockerhub does not look like a production quality base image.

Reply:
replaced with an alternative approach.